### DISCUSSION TITLE
GitHub Student Developer Pack in Ukraine

### DISCUSSION THREAD

**Discussion by GameHackerTV** [community/community#53541] [GitHub Education] [answered]
Hello everyone, I'm studying in Ukraine in a higher school (grades 10-11) how can I apply for a GitHub Student Developer Pack?
Who knows what documents I can provide from the Ukrainian school, and how can I translate them into English???

**Comment by samus-aran** [comment, marked as answer]
Hi @GameHackerTV 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍

**Reply by GameHackerTV** [reply]
It turns out I can get a certificate from the school. Is it stamped with my name, grade and school where I go to school? And also a question about "translate the document into English" - can I make a copy of the document and translate by hand the necessary data into English or is something else required? (And take a picture of them side by side)

**Reply by samus-aran** [reply]
> And also a question about "translate the document into English" - can I make a copy of the document and translate by hand the necessary data into English or is something else required? (And take a picture of them side by side)

Yes, absolutely 👍 Ensure the important information is translated.

**Reply by GameHackerTV** [reply]
Thank you very much, I was able to confirm.

**Reply by samus-aran** [reply]
That's awesome @GameHackerTV! 🎉 Thanks so much for coming back and sharing the update. This will help other community members seeking similar answers. Start enjoying GitHub Global Campus 🌟:

- Connect with a local Campus Expert. For more information on campus experts, see "[About Campus Experts](https://education.github.com/students/experts)."
- Explore and claim offers for free industry tools from the [Student Developer Pack](https://education.github.com/pack).
- See upcoming in-person and virtual events for students, curated by GitHub Education and student leaders.
- View assignments from [GitHub Classroom](https://classroom.github.com/) with upcoming due dates.
- Stay in the know with [Campus TV](https://www.twitch.tv/githubeducation) episodes. Campus TV is created by GitHub and student community leaders and can be watched live or on demand.
- Discover and contribute to student-created repositories from GitHub Community Exchange. For more information, see "[About GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange)."

What benefits are you excited to try out first? ✨

**Reply by GameHackerTV** [reply]
Student recruitment is excellent. Gives a lot of opportunities to develop, and improve your skills.

**Comment by GameHackerTV** [comment]
Lastly, if you are faced with getting a student pack, take a picture of the regular document and a copy of it in English. To be verified faster and without problems - take a photo with the camera on the website when you fill out the application. (uploaded photos from a phone have less credibility)

P.S. Special thanks to @samus-aran for the help <3

Have a good day and good luck to everyone

**Reply by lusparkl** [reply]
> Lastly, if you are faced with getting a student pack, take a picture of the regular document and a copy of it in English. To be verified faster and without problems - take a photo with the camera on the website when you fill out the application. (uploaded photos from a phone have less credibility)
> 
> P.S. Special thanks to @samus-aran for the help <3
> 
> Have a good day and good luck to everyone

How  you did a copy of the document in English?

**Reply by GameHackerTV** [reply]
@lusparkl I had to use Google Translate photo translation. And send them this photo with the original document lying next to it (next to the translated photo).
